# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../watchdog/no_sys_siglist.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- watchdog-5.4/src/fstab.c.vanilla	2008-11-27 18:45:20.000000000 +0100
+++ watchdog-5.4/src/fstab.c	2008-11-27 18:46:36.000000000 +0100
@@ -239,7 +239,11 @@
 /* Ensure that the lock is released if we are interrupted.  */
 static void
 handler (int sig) {
+    #if HAVE_DECL_SYS_SIGLIST
      die (EX_USER, "%s", sys_siglist[sig]);
+    #else
+     die (EX_USER, "signal %d",sig);
+    #endif
 }
 
 static void
